.github/workflows/build.yaml

```diff
@@ -9,9 +9,13 @@ jobs:
     steps:
     - name: deps
       run: |
+        sudo apt-get update
+        sudo apt-get install -y wget
+        echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable bionic main" > /etc/apt/sources.list.d/gazebo-stable.list
+        wget https://packages.osrfoundation.org/gazebo.key -O - | apt-key add -
         sudo apt update
```

Comment:
```suggestion
```

Reply:
Sorry ignore this suggestion. We will need to update after adding the new gazebo sources list.